starter-code/routes/index.js

```diff
@@ -6,4 +6,21 @@ router.get('/', (req, res, next) => {
   res.render('index');
 });
 
+
+router.get('/private', (req, res, next) => {
+  if(req.session.user) {
+    res.render('auth/private')
+  } else {
+    res.redirect('/auth/login')
+  }
+})
+
+router.get('/main', (req, res, next) => {
+  if(req.session.user) {
+    res.render('auth/mainpage')
+  } else {
+    res.redirect('/auth/login')
+  }
+})
+
```

Comment:
Nice work! Another way to write this out would be by making a middleware function like so 
```
const auth = (req, res, next) => {
  if (!req.session.user) {
    res.redirect("/auth/login");
    return;
  }
  next();
};

router.get("/private", auth, (req, res, next) => {
  res.render("auth/private");
});

router.get("/main", auth, (req, res, next) => {
  res.render("auth/mainpage");
});